keystore/filesystem/server_keystore.go

```diff
@@ -75,8 +75,9 @@ func newFilesystemKeyStore(privateKeyFolder, publicKeyFolder string, encryptor k
 		return nil, err
 	}
 	fi, err := os.Stat(directory)
-	if nil == err && runtime.GOOS == "linux" && fi.Mode().Perm().String() != "-rwx------" {
-		log.Errorln("Key store folder has an incorrect permissions")
+	const expectedPermission = "-rwx------"
+	if nil == err && runtime.GOOS == "linux" && fi.Mode().Perm().String() != expectedPermission {
+		log.Errorf("Key store folder has an incorrect permissions, expected: %s", expectedPermission)
```

Comment:
It may be useful to log actual permissions as well. Like we do in utils/utils.go for files.